packages/components/src/SectionMessage/SectionMessage.tsx

```diff
@@ -116,10 +131,10 @@ export const SectionMessage = ({
           {Icon && <Icon />}
         </div>
         {closeButton && (
-          <button
+          <Button
```

Comment:
`useButton` hook might be more appropiate here, would not require to override all button styles then.